modules/aws/vpc_test.go

```diff
@@ -17,6 +21,37 @@ func TestGetDefaultVpc(t *testing.T) {
 	assert.Regexp(t, "^vpc-[[:alnum:]]+$", vpc.Id)
 }
 
+func TestGetVpcById(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	vpc := createVpc(t, region)
+	defer deleteVpc(t, *vpc.VpcId, region)
+
+	vpcTest := GetVpcById(t, *vpc.VpcId, region)
+	assert.Equal(t, *vpc.VpcId, vpcTest.Id)
+}
+
+func TestGetVpcsE(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	azs := GetAvailabilityZones(t, region)
+
+	var isDefaultFilterName = "isDefault"
+	var isDefaultFilterValue = "true"
```

Comment:
Canonical way to declare and initialize vars in golang is `:=`. E.g `isDefaultFilterValue := "true"`